src/libostree/ostree-sysroot-deploy.c

```diff
@@ -2866,6 +2929,17 @@ sysroot_finalize_deployment (OstreeSysroot     *self,
         return FALSE;
     }
 
+  static const gchar * const SEMODULE_CMD[] = {
+    "semodule", "-N", "--rebuild-if-modules-changed"
+  };
+  gint exit_status;
+  if (!run_in_deployment(deployment_dfd, SEMODULE_CMD,
+                         sizeof(SEMODULE_CMD) / sizeof(*SEMODULE_CMD),
+                         &exit_status, error))
+    return FALSE;
+  if (!g_spawn_check_exit_status(exit_status, NULL))
+    g_message ("Failed to refresh SELinux policy - the policy contents may be inconsistent");
```

Comment:
I guess you made this non-fatal to handle ratcheting it in? Let's figure out instead how to auto-detect it so we can tighten this. Two obvious options:
- Feature detection at runtime: e.g. search for switch in output of `semodule --help`? This would be best I think but can get quite verbose.
- Compile-time version check: add autoconf goop to detect if `libselinux` is high enough? Hmm, it looks like on Fedora `semodule` is part of `policycoreutils`. But they're all versioned together, right?

Reply:
The second option won't work, because we depend on a given `semodule` version *inside* the deployment, which may not match the runtime where the `ostree` is running (may be an older/newer/completely different deployment).

The first option I had thought off, it just didn't seem worth it. But since you request it, I'll try to do it this way.

Reply:
I already went with scanning the --help output, which turned out to be not that verbose nor ugly. Please have a look and let me know if it works for you.

It's not entirely clear to me how it would work with the env variable, but the idea doesn't sound very appealing to me... Yet, it's not up to me so I'll submit to whatever you guys prefer :)